pyarr/base.py

```diff
@@ -0,0 +1,627 @@
+from datetime import datetime
+
+from .request_api import RequestAPI
+
+
+class BaseAPI(RequestAPI):
+    """Base functions in all Arr api's"""
```

Comment:
Capitalize API

Reply:
assume this was in relation to the comment, done